cmd/forwarder/config.go

```diff
@@ -31,8 +29,19 @@ type IssConfig struct {
 	Debug                     bool          `env:"LOG_ISS_DEBUG"`
 	TlsConfig                 *tls.Config
 	MetricsRegistry           metrics.Registry
-	tokenMap                  map[string]string
-	tokenMapOnce              sync.Once
+}
+
+type AuthConfig struct {
+	RedisUrl        string        `env:"REDIS_URL,required"`
+	RedisKey        string        `env:"REDIS_KEY",required`
+	RefreshInterval time.Duration `env:"TOKEN_REFRESH_INTERVAL,default=1m,strict"`
+	Tokens          string        `env:"TOKEN_MAP,required"`
```

Comment:
@sheax0r Is the plan to store the old `TOKEN_MAP` string format in Redis? If so, what's the motivation behind that, over storing user/token pairs?

Reply:
@jmervine the plan is not to store the old `TOKEN_MAP` format in redis. The plan is to store a redis hash using [HGETALL](https://redis.io/commands/hgetall). Each key in the hash is a username, and each value is a json array of passwords that are valid for the user. This allows us to store a data structure that is equivalent to the old `TOKEN_MAP` value, but requires less custom string-manipulation (json has libs to manipulate it; the `TOKEN_MAP` string requires custom code and custom escape sequences).

TL;DR: This format gives us the same power as `TOKEN_MAP`, but is easier to read and write, and leverages native redis capabilities.